pkg/applicationserver/applicationserver.go

```diff
@@ -1049,6 +1049,14 @@ func (as *ApplicationServer) handleJoinAccept(
 		return err
 	}
 
+	entity, err := as.endDeviceRegistry.Get(ctx, ids, []string{"attributes"})
+	if err != nil {
+		log.FromContext(ctx).WithError(err).
+			Warn("Failed to retrieve end device attributes on join-accept")
+	} else {
+		joinAccept.Attributes = entity.Attributes
+	}
```

Comment:
Where do the other fields come from?

Reply:
The other fields of the joinAccept message you mean? As far as I understand, from an AS redis device registry, but that one doesn't seem to have the attributes.